centos/centos-8.x/centos-8.3-hpc/install_mellanoxofed.sh

```diff
@@ -1,15 +1,24 @@
 #!/bin/bash
 set -ex
 
-MLNX_OFED_DOWNLOAD_URL=https://azhpcstor.blob.core.windows.net/azhpc-images-store/MLNX_OFED_LINUX-5.2-1.0.4.0-rhel8.3-x86_64.tgz
-TARBALL=$(basename ${MLNX_OFED_DOWNLOAD_URL})
+VERSION="5.8-1.0.1.1"
+$COMMON_DIR/write_component_version.sh "MOFED" $VERSION
+TARBALL="MLNX_OFED_LINUX-$VERSION-rhel8.3-x86_64.tgz"
+MLNX_OFED_DOWNLOAD_URL=https://content.mellanox.com/ofed/MLNX_OFED-${VERSION}/$TARBALL
 MOFED_FOLDER=$(basename ${MLNX_OFED_DOWNLOAD_URL} .tgz)
 
-$COMMON_DIR/download_and_verify.sh $MLNX_OFED_DOWNLOAD_URL "dd13ffa54524af9509e3de7288a863aa48ed7016bb3d15446523703020c41715"
+$COMMON_DIR/download_and_verify.sh $MLNX_OFED_DOWNLOAD_URL "370f49fd9190f046e1a41853e9f03508b57f1a29fa01f73099e564e6529a979d"
 tar zxvf ${TARBALL}
 
 KERNEL=( $(rpm -q kernel | sed 's/kernel\-//g') )
-KERNEL=${KERNEL[-1]}
+# Uncomment the lines below if you are running this on a VM
+# RELEASE=( $(cat /etc/centos-release | awk '{print $4}') )
+# yum install -y https://vault.centos.org/${RELEASE}/BaseOS/x86_64/os/Packages/kernel-devel-${KERNEL}.rpm
 yum install -y kernel-devel-${KERNEL}
-./${MOFED_FOLDER}/mlnxofedinstall --kernel $KERNEL --kernel-sources /usr/src/kernels/${KERNEL} --add-kernel-support --skip-repo
+./${MOFED_FOLDER}/mlnxofedinstall --kernel $KERNEL --kernel-sources /usr/src/kernels/${KERNEL} --add-kernel-support --skip-repo --skip-unsupported-devices-check --without-fw-update
```

Comment:
Could you tell me the reason of having `--skip-unsupported-devices-check --without-fw-update` ?

Reply:
Reference: https://docs.nvidia.com/networking/display/MLNXOFEDv581011/Installing+MLNX_OFED

--skip-unsupported-devices-check - If the driver detects unsupported cards on your system, it will abort the installation procedure. To avoid this, make sure to add --skip-unsupported-devices-check flag during installation.

--without-fw-update - Intended to avoid unwanted f/w updates to the devices. 

We are also using these flags to maintain consistency with the Ubuntu scripts